Sources/Commands/SwiftTool.swift

```diff
@@ -599,21 +599,35 @@ public class SwiftTool {
     func evaluateExtensions(graph: PackageGraph) throws -> [ResolvedTarget: [ExtensionEvaluationResult]] {
         do {
             // Configure the inputs to the extension evaluation.
-            // FIXME: These paths are still fairly preliminary.
+
+            // The `extensions` directory is inside the workspace's main data directory, and contains all temporary
```

Comment:
The changes in this section is cleanup of the paths under the intermediates directory.